cmd/fs-v1.go

```diff
@@ -76,7 +76,7 @@ func newFSObjects(storage StorageAPI) (ObjectLayer, error) {
 // Should be called when process shuts down.
 func (fs fsObjects) Shutdown() error {
 	// List if there are any multipart entries.
-	_, err := fs.storage.ListDir(minioMetaBucket, mpartMetaPrefix)
+	_, err := fs.storage.ListDir(minioMetaBucket, "multipart")
```

Comment:
> You can keep "multipart" constant so that we have a single place for this.

yes.. I wanted to discuss why we need to keep incomplete uploads data in FS after shutdown.. but let's fix the code and talk about this later.